guides/common/attributes.adoc

```diff
@@ -99,6 +99,7 @@ ifeval::["{build}" == "foreman"]
 :BaseURL: https://docs.theforeman.org/master/
 :ManagingHostsDocURL: {BaseURL}Managing_Hosts/index-foreman.html#
 :ConfiguringAnsibleDocURL: {BaseURL}Configuring_Ansible/index-foreman.html#
+:AdministeringDocURL: {BaseURL}Administering_Red_Hat_Satellite/index-foreman.html#
```

Comment:
I considered using the `AdministeringProjectDocURL` attribute, but it is too long so I decided to keep things short. The attribute tor the upcoming `Administering with Ansible` guide might use the `AdministeringWithAndibleDocURL` attribute.

Reply:
I am very positive about this, I just have not had spare time to fix the directories. This has been reported in #194 and let's track it there.

Reply:
Thanks!